lua/nvim-tree/actions/init.lua

```diff
@@ -393,7 +393,6 @@ function M.setup(opts)
   require("nvim-tree.actions.node.open-file").setup(opts)
   require("nvim-tree.actions.root.change-dir").setup(opts)
   require("nvim-tree.actions.fs.create-file").setup(opts)
-  require("nvim-tree.actions.fs.rename-file").setup(opts)
```

Comment:
Was that removed accidentally?

Reply:
That setup function was removed as it was no longer needed.